scripts/gen_chapters.js

```diff
@@ -0,0 +1,52 @@
+/* jshint node:true */
+"use strict";
+
+var _ = require('lodash'), 
+  path = require('path'),
+  fs = require('fs'),
+  map_config_path = process.argv[4] ? process.argv[4] :
+  path.join(__dirname, 'map_config.json'),
+  map_config = require(map_config_path),
+  dat = fs.readFileSync(map_config_path, 'utf8'),
+  map_data = {},
+  map_id = process.argv[2],
+  width = map_config.width,
+  tileWidth = map_config.tileWidth,
+  tileHeight = map_config.tileHeight,
+  mapLength = map_config.maps[0].length,
+  mapRepeat = map_config.maps[0].repeat,
+  bridgeLength = map_config.bridge.length,
```

Comment:
some variables are using camel case and some have underscores in them. Let's follow one style

Reply:
ok I will follow underscores